podcast.rss

```diff
@@ -1572,8 +1572,8 @@
       <description>
         <![CDATA[<p>Velicham Qur'an Dars series</p><p>Qur'an class # 107</p><p>💠Surah Al-Feel 💠<br />ആയത്ത്-1</p><p>ِ(أَلَمْ تَرَ كَيْفَ فَعَلَ رَبُّكَ بِأَصْحَابِ الْفِيلِ<br />ആനക്കാരോട്&zwnj; നിന്റെ രക്ഷിതാവ്&zwnj; എങ്ങനെ പ്രവർത്തിച്ചുവെന്ന് നീ കണ്ടില്ലേ?</p><p>🔸Admin Team Velicham Onlive🔸</p>]]>
       </description>
-      <enclosure url="https://velicham-onlive.github.io/audios/105-feel/7-vqd-podcast-unit105.m4a" type="audio/m4a" length="893000"/>
-      <itunes:duration>08:40</itunes:duration>
+      <enclosure url="https://velicham-onlive.github.io/audios/105-feel/7-vqd-podcast-unit105.mp3" type="audio/mp3" length="4072000"/>
```

Comment:
Incorrect name for new file, please correct

Reply:
the file name was changed according to the other files in the folder.(was previously named as 106.mp3 under unit 105).

Reply:
Good catch